src/main/kotlin/no/nav/brukerdialog/domenetjenester/mottak/PreprosesseringsService.kt

```diff
@@ -28,7 +28,7 @@ class PreprosesseringsService(
         val dokumentEier = DokumentEier(søkerFødselsnummer)
 
         logger.info("Genererer Oppsummerings-PDF...")
-        val oppsummeringPdf = pdfService.genererPDF(preprosesseringsData.pdfData)
+        val oppsummeringPdf = pdfService.genererPDF(preprosesseringsData.pdfData, true)
```

Comment:
Ville vært best om du ikke setter dette flagget til true i første omgang.
Dette vil påvirke pdf-generering i kafka streamene for de forskjellige søknadene.